### DISCUSSION TITLE
Entry for 易々 has the wrong sentence for the wrong reading

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#73] [Errors and Corrections]
易々 can be read both いい and やすやす apparently.

For the jitendex entry with the reading やすやす, we have no example sentence.

For the entry with the reading いい, we have the following sentence: どの仕事も**やすやす**とはできない。

I think this example sentence should be moved to the other entry instead.

**Comment by stephenmk** [comment]
Thanks. There seem to be many index strings in the database that are ambiguous; I come across mis-indexed sentences like this occasionally. I should write a program to search for all these ambiguities so that they can all be fixed. I might go ahead and do that right now while I'm thinking about it.

**Reply by stephenmk** [reply]
Unfortunately it seems like there are [a ton of them](https://github.com/JMdictProject/JMdictIssues/issues/121).

**Comment by JMdictProject** [comment]
FWIW I have now pinned all those 易々 cases to the 易々/やすやす entry.
